consensus/types/src/attestation.rs

```diff
@@ -412,13 +412,7 @@ impl<E: EthSpec> AttestationBase<E> {
     pub fn extend_aggregation_bits(
         &self,
     ) -> Result<BitList<E::MaxValidatorsPerSlot>, ssz_types::Error> {
-        let mut extended_aggregation_bits: BitList<E::MaxValidatorsPerSlot> =
-            BitList::with_capacity(self.aggregation_bits.len())?;
-
-        for (i, bit) in self.aggregation_bits.iter().enumerate() {
-            extended_aggregation_bits.set(i, bit)?;
-        }
-        Ok(extended_aggregation_bits)
+        self.aggregation_bits.resize::<E::MaxValidatorsPerSlot>()
```

Comment:
Snuck in this small change since the new `ssz_types` release has this function available